R/lasso-development.R

```diff
@@ -216,7 +214,7 @@ LassoDevelopment <- R6Class("LassoDevelopment",
       if (self$params$type == 'classification') {
         private$fitLogit <- glm(
           as.formula(paste(self$params$predictedCol, '.', sep = " ~ ")),
-          data = private$dfTrain,
+          data = private$dfTrainRaw,
```

Comment:
How come we're fitting the model using the dummified data? Original intent of dfTrainRaw was just for the top-3 factor calculations.

This might be why the code is running more slowly. When fitting models, R does the dummification automatically.